app/models/ability.rb

```diff
@@ -0,0 +1,10 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return unless user.present?
+
+    can :manage, Expense, author: user
+    can :manage, Group, user:
```

Comment:
- [ ] Initially, I was unable to run your app locally as a result of this syntax error. Kindly fix this error. From your schema, I think you meant `:user`.

  ![Screenshot_2022-04-19_15-46-12](https://user-images.githubusercontent.com/36542986/164031642-79c085bb-7665-43ec-88af-eb98c0bf329d.png)

Reply:
Thanks for spotting that